.github/workflows/links-check.yml

```diff
@@ -33,3 +33,10 @@ jobs:
           args: --offline --verbose --no-progress './**/*.md'
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: External Link Checker
+        uses: JustinBeckwith/linkinator-action@af984b9f30f63e796ae2ea5be5e07cb587f1bbd9 # v2.3
+        with:
+          paths: "**/*.md"
+          retry: true
+          skip: "https://(?:www.|)github.com/fastify/.\\*/(?:commit|issues|compare)/, http://0.0.0.0, http://127.0.0.1, http://localhost"
```

Comment:
Linkinator doesn't like use of `http://localhost`, so this needs removing. The entire skip section could be dropped tbh.